docs/pages/database-access/guides/aws-opensearch.mdx

```diff
@@ -0,0 +1,349 @@
+
+![Mapped User](../../../img/database-access/guides/aws-opensearch/02-opensearch_mapped_users.png)
+
+Configure IAM role mapping and add OpenSearch Mange Cluster access ARN as a users:
+
+![IAM Role mapping](../../../img/database-access/guides/aws-opensearch/03-opensearch_iam_role_mapping.png)
+
+Finally, click the Map button to apply the settings.
+
+## Step 2/4. Configure the Teleport IAM role mapping
+
+The next step is to give your Teleport users permissions to assume AWS IAM roles
+when accessing AWS resources through your Teleport cluster.
+
+You can do this by creating a Teleport role with the `db_users` field
+listing the IAM role ARN created in the previous step. Create a file called
+`aws-opensearch-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: aws-opensearch-access
+spec:
+  allow:
+    db_labels:
+      'env': 'dev'
+    db_users:
+    - 'ExampleTeleportOpenSearchRole'
+```
+
+Create the new role:
+
+```code
+$ tctl create -f aws-opensearch-access.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="aws-opensearch-access"!)
+
+## Step 3/4. Install the Teleport Database Service
+
+Create an EC2 instance to host the Teleport Database Service, and attach the
+`TeleportDatabaseService` AWS IAM role to it. If you're hosting the service another
+way, you must provide AWS credentials to the service - see [AWS credentials
+configuration](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html)
+for more details.
+
+<Admonition type="warning" title="non-standard AWS regions">
+For non-standard AWS regions such as AWS GovCloud (US) regions and AWS China
+regions, please set the corresponding region in the `AWS_REGION` environment
+variable or in the AWS credentials file so that the Database Service can use
+the correct STS endpoint.
+</Admonition>
+
+### Generate a token
+
+<Details title="Alterative methods">
+
+For users with a lot of infrastructure in AWS, or who might create or recreate
+many instances, consider alternative methods for joining new EC2 instances running
+Teleport:
+
+- [Configure Teleport to Automatically Enroll EC2 instances (Preview)](../../server-access/guides/ec2-discovery.mdx)
+- [Joining Nodes via AWS IAM
+  Role](../../management/join-services-to-your-cluster/aws-iam.mdx)
+- [Joining Nodes via AWS EC2 Identity Document](../../management/join-services-to-your-cluster/aws-ec2.mdx)
+
+</Details>
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Use the token provided by the output of this command in the next step.
+
+### Install and start Teleport
+
+Install Teleport on the host where you will run the Teleport Database
+Service. See our [Installation](../../installation.mdx) page for options
+besides Linux servers.
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<Tabs>
+<TabItem label="Using a config file">
```

Comment:
@smallinsky in this tab and "Static Config", both use a a local teleport config file, right? The only difference is that one creates a new file from scratch, and the other provides only the `db_service` section of that config? If so, I suggest we're more clear about this distinction.

Reply:
Yes, They are quite the same with a difference that the "Using a config file" produce the while teleport file thatand   requires a `teleport db configure create` call where Static Config omits this step and guide a user how to edit existing config to just add opensearch database to the already exited configuration. 

I have diffrcient this steps as "Generating a new Teleport Config file" and "Modifying the existing Database Service Config"